modules/format_strawberryfield_views/src/Plugin/views/filter/AdvancedSearchApiFulltext.php

```diff
@@ -578,15 +578,18 @@ public function buildExposedForm(&$form, FormStateInterface $form_state) {
         '#default_value' => $form_state->getValue($searched_fields_identifier),
       ];
 
-      if (isset($form[$this->options['id'] . '_wrapper'])) {
-        $form[$this->options['id'] . '_wrapper'][$searched_fields_identifier]
-          = $newelements[$searched_fields_identifier];
+      if (empty($form[$this->options['id'] . '_wrapper'])) {
+        $form[$this->options['id'] . '_wrapper'] =
+          [
```

Comment:
@patdunlavey small thing, can you move the `[` to the previous line? DCS

Reply:
This line is removed by using the buildValueWrapper method (see next)